src/main/java/dev/snowdrop/release/services/GitService.java

```diff
@@ -139,4 +141,39 @@ public void createGithubIssue(final String mdText, final String issueTitle, fina
         githubIssueBuilder.label(label);
         githubIssueBuilder.create();
     }
+
+    /**
+     * Close issues
+     * @param label
+     * @param token
+     * @param repoName
+     * @param version
+     * @throws IOException
+     */
+    public void closeOldCveIssues(final String label, final String token, final String repoName, final String version) throws IOException {
+        final String issueTitle = this.getCveIssueTitle(version);
+        GitHub github = new GitHubBuilder().withOAuthToken(token).build();
+        GHRepository gitHubRepo = github.getRepository(repoName);
+        List<GHIssue> ghIssues = gitHubRepo.getIssues(GHIssueState.OPEN);
```

Comment:
There's no other way than to retrieve all the open issues and then filter them out?

Reply:
Not that I could find out. I can only see 2 methods for querying issues, one with the state and the other with the state and the milestone.